boards/arm64/qemu/qemu-a53/configs/nsh/defconfig

```diff
@@ -11,7 +11,6 @@ CONFIG_ARCH_BOARD="qemu-a53"
 CONFIG_ARCH_BOARD_QEMU_A53=y
 CONFIG_ARCH_CHIP="qemu"
 CONFIG_ARCH_CHIP_QEMU=y
-CONFIG_ARCH_CHIP_QEMU_A53=y
```

Comment:
@HidenoriMatsubayashi need change Kconfig too, I think.

Reply:
Thanks. Fixed.

Reply:
By the way, I think we should also change the name `CONFIG_ARCH_BOARD_QEMU_A53` (+ directory name). Shall I respond with another PR?

Reply:
I think we can fix in one patch.

Reply:
OK, I'll fix it.